json-data-on-the-web.ipynb

```diff
@@ -0,0 +1,901 @@
+       "    <tr>\n",
+       "      <th>quasiben/fiftyfizzbuzzes/master</th>\n",
+       "      <td>536</td>\n",
+       "    </tr>\n",
+       "    <tr>\n",
+       "      <th>numba/numba-examples/master</th>\n",
+       "      <td>462</td>\n",
+       "    </tr>\n",
+       "    <tr>\n",
+       "      <th>geoscixyz/gpgLabs/master</th>\n",
+       "      <td>370</td>\n",
+       "    </tr>\n",
+       "    <tr>\n",
+       "      <th>dask/dask-examples/master</th>\n",
+       "      <td>347</td>\n",
+       "    </tr>\n",
+       "    <tr>\n",
+       "      <th>nteract/examples/master</th>\n",
+       "      <td>347</td>\n",
+       "    </tr>\n",
+       "    <tr>\n",
+       "      <th>data-8/textbook/gh-pages</th>\n",
+       "      <td>333</td>\n",
+       "    </tr>\n",
+       "    <tr>\n",
+       "      <th>SoftmatterLMU-RaedlerGroup/notebooks_e3_ws1819/master</th>\n",
+       "      <td>329</td>\n",
+       "    </tr>\n",
+       "  </tbody>\n",
+       "</table>\n",
+       "</div>"
+      ],
+      "text/plain": [
+       "                                                     spec\n",
+       "ipython/ipython-in-depth/master                     32270\n",
+       "jupyterlab/jupyterlab-demo/master                    8013\n",
+       "ines/spacy-io-binder/live                            2798\n",
+       "binder-examples/r/master                             1765\n",
+       "bokeh/bokeh-notebooks/master                         1640\n",
+       "DS-100/textbook/master                               1239\n",
+       "mkozturk/CMPE140/master                              1037\n",
+       "rationalmatter/juno-demo-notebooks/master             994\n",
+       "QuantStack/xeus-cling/stable                          977\n",
+       "marlohaering/SE1-Notebook/master                      841\n",
+       "RasaHQ/rasa_core/master                               635\n",
+       "binder-examples/julia-python/master                   555\n",
+       "QISKit/qiskit-tutorial/master                         554\n",
+       "quasiben/fiftyfizzbuzzes/master                       536\n",
+       "numba/numba-examples/master                           462\n",
+       "geoscixyz/gpgLabs/master                              370\n",
+       "dask/dask-examples/master                             347\n",
+       "nteract/examples/master                               347\n",
+       "data-8/textbook/gh-pages                              333\n",
+       "SoftmatterLMU-RaedlerGroup/notebooks_e3_ws1819/...    329"
+      ]
+     },
+     "execution_count": 15,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "df.spec.value_counts().nlargest(20).to_frame().compute()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Persist in memory\n",
+    "\n",
+    "This dataset fits nicely into memory. Lets avoid pulling from the cloud every time we do an operation and instead keep data local."
```

Comment:
I know mybinder.org is on the cloud, but maybe you should just say, "avoid pulling from mybinder.org"?

Reply:
Done